package.json

```
@@ -9,7 +9,9 @@
     "ng": "ng",
     "start": "ng serve",
     "start-dev": "ng build --watch",
-    "build-prod": "node --max_old_space_size=8048 node_modules/@angular/cli/bin/ng build --prod --aot -c=production",
+    "build-volta": "node --max_old_space_size=8048 node_modules/@angular/cli/bin/ng build --prod --aot -c=volta",
```

This not gonna work. You do not have volta config file.
```suggestion
    "build-volta": "node --max_old_space_size=8048 node_modules/@angular/cli/bin/ng build --prod --aot -c=production",
```
It doesn't matter what configuration file is.